bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/BigtableDataGrpcClient.java

```diff
@@ -284,4 +299,10 @@ public void onClose(Status status, Metadata trailers) {
       }
     };
   }
+
+  private void returnToPool(final Channel channel) {
+    if (channel != channelPool) {
```

Comment:
This is better, please add a comment saying that when channel is not channelPool, we are using a PooledChannel.